src/app/shared/components/login/login-form/login-form.component.spec.ts

```diff
@@ -21,8 +23,19 @@ describe('Login Form Component', () => {
     accountFacade = mock(AccountFacade);
 
     await TestBed.configureTestingModule({
-      declarations: [LoginFormComponent, MockComponent(ErrorMessageComponent), MockComponent(InputComponent)],
-      imports: [ReactiveFormsModule, RouterTestingModule, TranslateModule.forRoot()],
+      declarations: [LoginFormComponent, MockComponent(ErrorMessageComponent)],
+      imports: [
+        FormlyModule.forRoot({
```

Comment:
Don't use the FormlyTestingComponentsModule, just use the FormlyTestingModule. It includes a preconfigured FormlyModule and the ReactiveFormsModule. FormlyTestingComponentsModule is for testing types, wrappers etc